src/constant_valid_intervals.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+
+from src.constants_json_strings import *
+
+VALID_INTERVAL = "Valid interval"
+VALID_TYPE = "Valid type"
+PARAMETER_EXAMPLE = "Example"
+PARAMETER_DEFINITION = "Definition"
+DEFAULT = "Default value"
+def extend_definitions(parameter_definitions, parameter, valid_interval, valid_type, example, definition, default):
+
+    dict_parameter = {
+        parameter: {VALID_INTERVAL: valid_interval,
+                    VALID_TYPE: valid_type,
+                    PARAMETER_EXAMPLE: example,
+                    PARAMETER_DEFINITION: definition,
+                    DEFAULT: default}
+    }
+    parameter_definitions = parameter_definitions.append(pd.DataFrame.from_dict(dict_parameter))
```

Comment:
So if I understand it right, you turn a dict into a single row dataframe and you append this single row dataframe into a list (inside parenthesis: `parameter_definitions` seems to be a list at first because you use the builtin `append` Why do you provide the return value of `.append()` to the list `parameter_definitions` as using `.append()` will already fill the purpose to append a new value to the `parameter_definitions` list?)

Why don't you use directly a dataframe with columns
```
                    VALID_INTERVAL,
                    VALID_TYPE,
                    PARAMETER_EXAMPLE,
                    PARAMETER_DEFINITION,
                    DEFAULT,
```

Reply:
My bad, `parameter_definitions` is indeed a dataframe